frontend/src/metabase/components/LastEditInfoLabel.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import styled from "styled-components";
+import { t } from "ttag";
+import moment from "moment";
+
+import { color } from "metabase/lib/colors";
+
+import { getUser } from "metabase/selectors/user";
+
+const Label = styled.span`
+  font-weight: bold;
+  color: ${color("text-medium")};
+`;
+
+function mapStateToProps(state) {
+  return {
+    user: getUser(state),
+  };
+}
+
+LastEditInfoLabel.propTypes = {
+  item: PropTypes.shape({
+    "last-edit-info": PropTypes.shape({
+      id: PropTypes.number.isRequired,
+      email: PropTypes.string.isRequired,
+      first_name: PropTypes.string.isRequired,
+      last_name: PropTypes.string.isRequired,
+      timestamp: PropTypes.string.isRequired,
+    }).isRequired,
+  }),
+  user: PropTypes.shape({
+    id: PropTypes.number,
+  }).isRequired,
+};
+
+function formatEditorName(firstName, lastName) {
+  const lastNameFirstLetter = lastName.charAt(0);
+  return `${firstName} ${lastNameFirstLetter}.`;
+}
+
+function LastEditInfoLabel({ item, user, ...props }) {
+  const { first_name, last_name, id: editorId, timestamp } = item[
+    "last-edit-info"
+  ];
+  const time = moment(timestamp).fromNow();
+
+  const editor =
+    editorId === user.id ? "you" : formatEditorName(first_name, last_name);
```

Comment:
I wonder how will this now work in translations. If I understand this correctly
```
{t`Edited ${time} by ${editor}`}
```
Will receive the first name and last name of the person who last edited something (if that person is not the currently logged in user) and will also mark that as a string for translation?

Reply:
Yes, the same way it's done in the reference section (see [here](https://github.com/metabase/metabase/blob/e736e6b2956b14074ea37e0d70accb724812075d/frontend/src/metabase/reference/metrics/MetricQuestions.jsx#L99))

Reply:
That feels a bit weird.
I would've expected some kind of a placeholder for the name.

But this has nothing to do with your changes. Great work so far. Thanks for addressing requested changes.